root/user/components/UserInlineList.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
```

Comment:
You might want to add your own copyright notice here, or replace the current one, or not change anything. Note that this notice will be public and indexed by search engines.